packages/dicomImageLoader/src/shared/decoders/decodeJPEG2000.ts

```diff
@@ -1,20 +1,22 @@
 // https://emscripten.org/docs/api_reference/module.html
-import openJpegFactory from '@cornerstonejs/codec-openjpeg/dist/openjpegwasm_decode.js';
+import openJpegFactory from '@cornerstonejs/codec-openjpeg/dist/openjpegwasm_decode';
```

Comment:
Ideally these should use the exports key value rather than assuming a dist/... path as that allows the paths to be updated/changed, whereas this is a fixed path.